<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCRDGroupDialogUI</class>
 <widget class="QDialog" name="NCRDGroupDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Groups</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="layList">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QListView" name="listGroups"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="layBtn">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="NCRDPanelButtons" name="panelButtons" native="true"/>
           </item>
           <item>
            <spacer name="sp2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabGroupProp">
       <attribute name="title">
        <string>Group settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="gbGroup">
          <property name="title">
           <string>Identify</string>
          </property>
          <layout class="QGridLayout" name="layGroupProp" columnstretch="1,5,1">
           <property name="spacing">
            <number>4</number>
           </property>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Group expression</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="groupID"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Group ID</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QToolButton" name="btnExpBuild">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="wGroupExp"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbHF">
          <property name="title">
           <string>Header and Footer</string>
          </property>
          <layout class="QGridLayout" name="layHdrFtr">
           <item row="0" column="0">
            <widget class="QCheckBox" name="cbHeader">
             <property name="text">
              <string>Group header</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="spHeader">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="cbFooter">
             <property name="text">
              <string>Group footer</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDoubleSpinBox" name="spFooter">
             <property name="suffix">
              <string> mm</string>
             </property>
             <property name="maximum">
              <double>999.990000000000009</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QCheckBox" name="cbReprint">
             <property name="text">
              <string>Reprint header on new page</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbBreak">
          <property name="title">
           <string>Page break</string>
          </property>
          <layout class="QVBoxLayout" name="layPbreak">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Break condition (logical expression)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pageBreakCond"/>
           </item>
           <item>
            <widget class="QCheckBox" name="cbPrintOnNewPage">
             <property name="text">
              <string>Always starts on new page</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Reset variables</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="listVars"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NCRDPanelButtons</class>
   <extends>QWidget</extends>
   <header>ncrdpanelbuttons.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>listGroups</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>groupID</tabstop>
  <tabstop>wGroupExp</tabstop>
  <tabstop>btnExpBuild</tabstop>
  <tabstop>cbHeader</tabstop>
  <tabstop>spHeader</tabstop>
  <tabstop>cbFooter</tabstop>
  <tabstop>spFooter</tabstop>
  <tabstop>cbReprint</tabstop>
  <tabstop>pageBreakCond</tabstop>
  <tabstop>cbPrintOnNewPage</tabstop>
  <tabstop>listVars</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NCRDGroupDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NCRDGroupDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
